rustc nightly error on use std::intrinsics::copy_nonoverlapping;

Error compiling with
`rustc 1.64.0-nightly (c2f428d2f 2022-07-14)` 

Text of the error
 
```
error[E0658]: use of unstable library feature 'core_intrinsics': intrinsics are unlikely to ever be stabilized, instead they should be used through stabilized interfaces in the rest of the standard library
 --> /home/user/.cargo/registry/src/github.com-1ecc6299db9ec823/inplace_it-0.3.3/src/guards/slice_memory_guard.rs:7:5
  |
7 | use std::intrinsics::copy_nonoverlapping;
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: add `#![feature(core_intrinsics)]` to the crate attributes to enable
```
Following the recommendation to ignore the error by adding `#![feature(core_intrinsics)]`  to  `lib.rs` allows the code to compile again.  I have no insight on whether this is a wise choice, but it works.

--- a/src/guards/slice_memory_guard.rs
+++ b/src/guards/slice_memory_guard.rs
@@ -3,7 +3,7 @@ use core::{
     mem::{MaybeUninit, transmute},
     ptr::{drop_in_place, write},
 };
-use std::intrinsics::copy_nonoverlapping;
+use std::ptr::copy_nonoverlapping;
 
 /// Guard-struct used for correctly initialize uninitialized memory and `drop` it when guard goes out of scope.
 /// Usually, you *should not* use this struct to handle your memory.

